developers/wallets/index.md

```diff
@@ -21,37 +21,21 @@ CHAPI integrates easily into digital wallet software, allowing your wallet to re
 
 * [Working with `QueryByExample` format requests](querybyexample)
 - **Example Code**: the [chapi-demo-wallet](https://github.com/credential-handler/chapi-demo-wallet) contains a full example implementation and is referenced throughout this guide.
-- **Polyfill Library**: The [credential-handler-polyfill](https://github.com/credential-handler/credential-handler-polyfill) library provides the needed functionality in the browser.
+- **Polyfill Library**: The [credential-handler-polyfill](https://github.com/credential-handler/credential-handler-polyfill) library provides the needed `CredentialHandler` API within the browser.
 - **Helper Library**: The [web-credential-handler](https://github.com/credential-handler/web-credential-handler) library provides helper functions for CHAPI integration in your code.
 
-## Wallet Registration
+## Web Wallet Registration
 
-Wallets need to be registered with the browser as a Credential Handler in order to store or retrieve Verifiable Credentials on the Web.
+All Wallets need to be registered as a Credential Handler in the user's browser in order to store or retrieve Verifiable Credentials (VCs) from across the Web.
 
-### 1. Import the CHAPI Polyfill into your wallet app
-If you're developing on Node.js, add the credential-handler-polyfill library to your project.  You can also install the web-credential-handler helper library to simplify your code.
+A Web Wallet is typically made up of three main parts:
+* `/index.html` - main page for the wallet
+* `/manifest.json` - Web App Manifest containting credential handler affordances
+* `/wallet-worker.html` - routing endpoint for passing URLs between CHAPI and the Wallet
```

Comment:
To make it more clear which of these specifically named relative URLs *have* to be used and which are examples:

```suggestion
* A main page for the wallet (e.g., `/index.html`)
* Web App Manifest containing credential handler affordances (using `/manifest.json`)
* A routing endpoint for handling requests between CHAPI and the Wallet (e.g., `/wallet-worker.html`) 
```

Reply:
I rather liked the filename first approach as it gave me ready handles to hook onto later, but happy to swap them. I'll muse on it a bit.

Reply:
k...you talked me into it... 😉